doc/schema.md

```diff
@@ -0,0 +1,68 @@
+| Raw amplitude image            | IMG_RAMP     | 4     | Raw Amplitude Image                                                               |
+| Cartesian coordinate           | IMG_CART     | 8     | Cartesian coordinates (X,Y,Z) at each pixel considering camera at (0,0,0)          |
+| Unit vector                    | IMG_UVEC     | 16    | 3 channel Image showing unit vector at each pixel                                 |
+| Exposure time                  | EXP_TIME     | 32    | Time to capture phase data for a frame                                            |
+| Grayscale Image                | IMG_GRAY     | 64    | Ambient light image                                                               |
+| Illumination temperature       | ILLU_TEMP    | 128   | Temperature of the illumination unit of the camera                                |
+| Intrinsic calibration           | INTR_CAL     | 256   | Intrinsic calibration parameters                                                   |
+| Inverse intrinsic calibration  | INV_INTR_CAL | 512   | Inverse intrinsic calibration parameters                                           |
+| JSON model                     | JSON_MODEL   | 1024  | Application output in JSON format                                                 |
+| Distance Noise Image           | IMG_DIS_NOISE| 2048  | Distance noise image                                                              |
+
+The following code shows the configuration of the `FrameGrabber` object for radial distance and amplitude image:
+
+```cpp
+auto cam = std::make_shared<ifm3d::Camera>();
+auto fg = std::make_shared<ifm3d::FrameGrabber>(cam,ifm3d::IMG_RDIS|ifm3d::IMG_AMP);
+auto img = std::make_shared<ifm3d::ImageBuffer>();
+
+if (! fg->WaitForFrame(img.get(), 1000))
+    {
+      std::cerr << "Timeout waiting for camera!" << std::endl;
+      return -1;
+    }
+
+auto amp = img->AmplitudeImage();
+auto rdist = img->DistanceImage();
+
+```
+Passing `ifm3d::IMG_RDIS|ifm3d::IMG_AMP` as a parameter to `FrameGrabber` will configure the
+device connection to receive only radial distance and amplitude image. To obtain these images,
+call the corresponding funtion on container buffers as shown in code snippet. Calling
+a function for data or images which are not configured through the schema will result in empty buffers.
+
+Default value of schema is set to `ifm3d::IMG_AMP|ifm3d::IMG_CART` and can be overridden by
+passing the schema value to `Framegrabber` or by setting the env variable `IFM3D_MASK`.
+
+To get the schema value for a combination of data/images, add the decimal values of the required data/images
+and set the value to env variable IFM3D_MASK.
+
+For instance, setting a mask for the radial distance, amplitude and cartesian images ( 1+ 2+ 8 = 11):
+
+```
+on windows
+
+> set IFM3D_MASK = 11
+
+on linux
+
+$ export IFM3D_MASK = 11
+```
+
```

Comment:
From a C++ developer point of view the whole concept behind the environment variables sound quite awkward and pretty use-less. Things are really making sense when it comes to the command line tooling. So I recommend to mention the command line use-case with examples here will make things more obvious.